service/src/main/java/org/cbioportal/service/util/MutationMapperUtils.java

```diff
@@ -31,7 +47,60 @@
             structuralVariant.setSite1Position(fusion.getStartPosition().intValue());
             structuralVariant.setComments(fusion.getKeyword());
             structuralVariant.setNcbiBuild(fusion.getNcbiBuild());
+            structuralVariant.setVariantClass(fusion.getMutationType());
+            structuralVariant.setEventInfo(fusion.getProteinChange());
+            if (fusion.getProteinChange() != null) {
+                String proteinChange = fusion.getProteinChange();
+                // only parse proteinChange when its not Fusion or SV
+                if (!(proteinChange.equalsIgnoreCase("Fusion") || proteinChange.equalsIgnoreCase("SV"))) {
```

Comment:
When a fusion is carried within a mutation record, the protein change contains the SV event.

Reply:
Not sure if it's a SV event but the protein change is `SV`. Here is an example https://www.cbioportal.org/patient?studyId=nbl_broad_2013&caseId=TARGET-30-PANRRW